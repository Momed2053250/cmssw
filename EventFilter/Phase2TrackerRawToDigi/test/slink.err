%MSG-i AlpakaService:  (NoModuleName) 28-May-2025 12:24:33 CEST pre-events
AlpakaServiceSerialSync succesfully initialised.
Found 1 device:
  - Intel(R) Xeon(R) Silver 4114 CPU @ 2.20GHz
%MSG
%MSG-i CUDAService:  (NoModuleName) 28-May-2025 12:24:33 CEST pre-events
CUDA runtime version 12.8, driver version 12.4, NVIDIA driver version 550.163.01
CUDA device 0: Tesla V100-PCIE-32GB (sm_70)
%MSG
%MSG-i AlpakaService:  (NoModuleName) 28-May-2025 12:24:33 CEST pre-events
AlpakaServiceCudaAsync succesfully initialised.
Found 1 device:
  - Tesla V100-PCIE-32GB
%MSG
28-May-2025 12:24:38 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/relval/CMSSW_14_0_0_pre2/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_133X_mcRun4_realistic_v1_STD_2026D98_PU200_RV229-v1/2580000/0b2b0b0b-f312-48a8-9d46-ccbadc69bbfd.root
28-May-2025 12:25:30 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/relval/CMSSW_14_0_0_pre2/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_133X_mcRun4_realistic_v1_STD_2026D98_PU200_RV229-v1/2580000/0b2b0b0b-f312-48a8-9d46-ccbadc69bbfd.root
28-May-2025 12:25:32 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/relval/CMSSW_14_0_0_pre2/RelValTTbar_14TeV/GEN-SIM-DIGI-RAW/PU_133X_mcRun4_realistic_v1_STD_2026D98_PU200_RV229-v1/2580000/0b2b0b0b-f312-48a8-9d46-ccbadc69bbfd.root
----- Begin Fatal Exception 28-May-2025 12:25:32 CEST-----------------------
An exception of category 'StdException' occurred while
   [0] Constructing the EventProcessor
   [1] Constructing module: class=Phase2RawToClusterProducer@alpaka label='Unpacker'
Exception Message:
A std::exception was thrown.
/cvmfs/cms.cern.ch/el9_amd64_gcc12/external/alpaka/1.2.0-ce4b840cb6308c9be80d192f663d60d7/include/alpaka/queue/cuda_hip/QueueUniformCudaHipRt.hpp(50) 'TApi::setDevice(m_dev.getNativeHandle())' returned error  : 'cudaErrorInvalidDevice': 'invalid device ordinal'!
----- End Fatal Exception -------------------------------------------------
