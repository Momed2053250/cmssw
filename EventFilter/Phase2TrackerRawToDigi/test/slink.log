000 (1007322.000.000) 05/28 12:21:41 Job submitted from host: <188.185.129.8:9618?addrs=188.185.129.8-9618+[2001-1458-d03-1--100-8a]-9618&alias=bigbird101.cern.ch&noUDP&sock=schedd_3241060_289e>
...
040 (1007322.000.000) 05/28 12:22:02 Started transferring input files
	Transferring to host: <128.142.241.90:9618?addrs=128.142.241.90-9618+[2001-1458-301-ce--100-1cf]-9618&alias=b9g47n5843.cern.ch&noUDP&sock=slot1_1_2219_3c73_10871>
...
040 (1007322.000.000) 05/28 12:22:02 Finished transferring input files
...
001 (1007322.000.000) 05/28 12:22:02 Job executing on host: <128.142.241.90:9618?addrs=128.142.241.90-9618+[2001-1458-301-ce--100-1cf]-9618&alias=b9g47n5843.cern.ch&noUDP&sock=startd_1904_9069>
	SlotName: slot1_1@b9g47n5843.cern.ch
	AvailableGPUs = { GPUs_GPU_d93dc087 }
	CondorScratchDir = "/pool/condor/dir_2223869"
	Cpus = 2
	Disk = 2048
	GPUs = 1
	GPUs_GPU_d93dc087 = [ GlobalMemoryMb = 32494; MaxSupportedVersion = 12040; DriverVersion = 12.4; ComputeUnits = 80; ECCEnabled = true; DeviceUuid = "d93dc087-0f06-a7ee-241e-d0afdf77a74e"; DeviceName = "Tesla V100-PCIE-32GB"; CoresPerCU = 64; ClockMhz = 1380.0; DevicePciBusId = "0000:07:00.0"; Capability = 7.0; Id = "GPU-d93dc087" ]
	Memory = 6000
...
006 (1007322.000.000) 05/28 12:22:08 Image size of job updated: 52
	0  -  MemoryUsage of job (MB)
	0  -  ResidentSetSize of job (KB)
...
040 (1007322.000.000) 05/28 12:22:08 Started transferring output files
...
040 (1007322.000.000) 05/28 12:22:08 Finished transferring output files
...
007 (1007322.000.000) 05/28 12:22:08 Shadow exception!
	Error from slot1_1@b9g47n5843.cern.ch: STARTER at 128.142.241.90 failed to send file(s) to <188.185.129.8:9618>: |Error: reading from file /pool/condor/dir_2223869/raw2clusters.root: (errno 2) No such file or directory; SHADOW failed to receive file(s) from <128.142.241.90:39617>
	263  -  Run Bytes Sent By Job
	5954  -  Run Bytes Received By Job
...
012 (1007322.000.000) 05/28 12:22:08 Job was held.
	Transfer output files failure at execution point slot1_1@b9g47n5843.cern.ch while sending files to access point bigbird101. Details: reading from file /pool/condor/dir_2223869/raw2clusters.root: (errno 2) No such file or directory
	Code 12 Subcode 2
...
000 (1007323.000.000) 05/28 12:23:53 Job submitted from host: <188.185.129.8:9618?addrs=188.185.129.8-9618+[2001-1458-d03-1--100-8a]-9618&alias=bigbird101.cern.ch&noUDP&sock=schedd_3241060_289e>
...
040 (1007323.000.000) 05/28 12:24:01 Started transferring input files
	Transferring to host: <128.142.241.90:9618?addrs=128.142.241.90-9618+[2001-1458-301-ce--100-1cf]-9618&alias=b9g47n5843.cern.ch&noUDP&sock=slot1_1_2219_3c73_10872>
...
040 (1007323.000.000) 05/28 12:24:01 Finished transferring input files
...
001 (1007323.000.000) 05/28 12:24:01 Job executing on host: <128.142.241.90:9618?addrs=128.142.241.90-9618+[2001-1458-301-ce--100-1cf]-9618&alias=b9g47n5843.cern.ch&noUDP&sock=startd_1904_9069>
	SlotName: slot1_1@b9g47n5843.cern.ch
	AvailableGPUs = { GPUs_GPU_d93dc087 }
	CondorScratchDir = "/pool/condor/dir_2224068"
	Cpus = 2
	Disk = 2048
	GPUs = 1
	GPUs_GPU_d93dc087 = [ GlobalMemoryMb = 32494; MaxSupportedVersion = 12040; DriverVersion = 12.4; ComputeUnits = 80; ECCEnabled = true; DeviceUuid = "d93dc087-0f06-a7ee-241e-d0afdf77a74e"; DeviceName = "Tesla V100-PCIE-32GB"; CoresPerCU = 64; ClockMhz = 1380.0; DevicePciBusId = "0000:07:00.0"; Capability = 7.0; Id = "GPU-d93dc087" ]
	Memory = 6000
...
006 (1007323.000.000) 05/28 12:24:15 Image size of job updated: 41624
	41  -  MemoryUsage of job (MB)
	41624  -  ResidentSetSize of job (KB)
...
006 (1007323.000.000) 05/28 12:25:32 Image size of job updated: 903628
	41  -  MemoryUsage of job (MB)
	41624  -  ResidentSetSize of job (KB)
...
040 (1007323.000.000) 05/28 12:25:32 Started transferring output files
...
040 (1007323.000.000) 05/28 12:25:32 Finished transferring output files
...
007 (1007323.000.000) 05/28 12:25:32 Shadow exception!
	Error from slot1_1@b9g47n5843.cern.ch: STARTER at 128.142.241.90 failed to send file(s) to <188.185.129.8:9618>: |Error: reading from file /pool/condor/dir_2224068/raw2clusters.root: (errno 2) No such file or directory; SHADOW failed to receive file(s) from <128.142.241.90:38055>
	2256  -  Run Bytes Sent By Job
	6106  -  Run Bytes Received By Job
...
012 (1007323.000.000) 05/28 12:25:32 Job was held.
	Transfer output files failure at execution point slot1_1@b9g47n5843.cern.ch while sending files to access point bigbird101. Details: reading from file /pool/condor/dir_2224068/raw2clusters.root: (errno 2) No such file or directory
	Code 12 Subcode 2
...
009 (1007322.000.000) 05/29 12:22:40 Job was aborted.
	Job removed by SYSTEM_PERIODIC_REMOVE due to wall time exceeded allowed max.

	Job terminated by itself at 2025-05-28T10:22:08Z (using method 0: OF_ITS_OWN_ACCORD).
...
009 (1007323.000.000) 05/29 12:25:42 Job was aborted.
	Job removed by SYSTEM_PERIODIC_REMOVE due to wall time exceeded allowed max.

	Job terminated by itself at 2025-05-28T10:25:32Z (using method 0: OF_ITS_OWN_ACCORD).
...
