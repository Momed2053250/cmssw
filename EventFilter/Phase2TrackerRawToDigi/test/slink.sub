# slink.submit
Universe       = vanilla

# use our wrapper script
Executable     = run_slink.sh

# pass along the cfg
transfer_input_files    = SLinkProducerAndUnpacker_cfg.py, run_slink.sh

# grab the same CMSSW env vars inside the job
getenv         = True

# only transfer output ROOT file back
should_transfer_files   = YES
when_to_transfer_output = ON_EXIT
transfer_output_files   = raw2clusters.root

+RequestGPUs           = 1
request_cpus           = 1
request_memory         = 4GB

# logs
log     = slink.log
output  = slink.out
error   = slink.err

# only run on machines with GPUs
#Requirements = (CUDADeviceCount >= 1)

# submit one job (or change to “queue 10” for 10 jobs)
queue 1

