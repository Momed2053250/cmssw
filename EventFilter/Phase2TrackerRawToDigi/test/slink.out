CondDBESSource Statistics
DataProxy 1 setInterval 0 Runs 0 Lumis 0 Refresh 0 Actual Refresh 0 Reconnect 0 Actual Reconnect 0
TrackerDetToDTCELinkCablingMapRcd / : frontier://FrontierProd/CMS_CONDITIONS, TrackerDetToDTCELinkCablingMap__OT800_IT711__T33__OTOnly
  
  Time look up, payloadIds:


